userwidgets/tiles.ratingTile/modules/require/ratingTileController.js

```diff
@@ -71,6 +71,11 @@ define(function() {
       });
         return animationObejct;
       },
+      /**
+         *	@function getPlatformSpecific
+         * 	@description This generic function is to return the animationConfig
+         * 	@private
+         */
       getPlatformSpecific:function(){
```

Comment:
The function name and functionality is contradicting. Please change the function name according to the task it is performing

Reply:
done